platform/darwin/src/MGLOfflineStorage.mm

```diff
@@ -357,6 +360,31 @@ - (void)addPackForRegion:(id <MGLOfflineRegion>)region withContext:(NSData *)con
         [[strongSelf mutableArrayValueForKey:@"packs"] addObject:pack];
         if (completion) {
             completion(pack, error);
+            
+            MGLTilePyramidOfflineRegion *tileRegion = MGL_OBJC_DYNAMIC_CAST(region, MGLTilePyramidOfflineRegion);
+            MGLShapeOfflineRegion *shapeRegion = MGL_OBJC_DYNAMIC_CAST(region, MGLShapeOfflineRegion);
+            
+            NSDictionary *attributes;
+            
+            if ([region isKindOfClass:[MGLTilePyramidOfflineRegion class]]) {
+                attributes = @{
+                  @"event": MMEventTypeOfflineDownloadStart,
+                  @"shapeForOfflineRegion": @"tileregion",
+                  @"minZoom": [NSNumber numberWithDouble:tileRegion.minimumZoomLevel],
+                  @"maxZoom": [NSNumber numberWithDouble:tileRegion.maximumZoomLevel],
+                  @"styleURL": region.styleURL
+                  };
+            } else {
+                attributes = @{
+                   @"event": MMEventTypeOfflineDownloadStart,
```

Comment:
All the attribute keys should be of the form `MMEEventKey...`; for example, instead of `event` use `MMEEventKeyEvent`.

You'll probably need to add constants for the other keys?